autotest.pm

```diff
@@ -54,7 +54,10 @@ loadtest is called.
 sub find_script {
     my ($script) = @_;
     my $casedir = $bmwqemu::vars{CASEDIR};
-    my $script_override_path = join('/', $bmwqemu::vars{PRJDIR} // '', 'factory/other', $script);
+    my $script_override_path = join('/', 'factory/other', $script);
+    # FIXME: This was previously relying on PRJDIR. Find a better way to reference a custom script
+    #        provided as an asset. This feature is likely superseded by openqa-clone-custom-git-refspec
+    #        anyways.
```

Comment:
There is still a valid use case. What I had in mind is the following: Imagine a discussion in a bug investigation when someone suggests "Could you please call these three lines of code in the openQA code to try out something or gather some data". Creating a fork of the test distribution and reference that in openqa-clone-custom-git-refspec is an additional hurdle that many are not willing to take. See https://github.com/os-autoinst/os-autoinst/pull/1019 and http://open.qa/docs/#_schedule_asset_nr_url for the original idea.

Reply:
I simply made it use `$bmwqemu::vars{ASSETDIR}` which is populated by the worker. That makes more sense than the previous code anyways because the logic for locating assets (and possibly caching them) is on the worker-side.